packages/monitoring-system/data-processor/src/data-processor-factory.ts

```diff
@@ -50,10 +53,10 @@ export class DataProcessorFactory implements Factory {
    */
   public getDataProcessor(task: Task): DataProcessor {
     switch (task) {
-      case Task.ProcessLogs:
+      case Task.ProcessCloudLogs:
         return new CloudLogsProcessor(this.getLogsProcessorOptions());
-      case Task.ProcessGCF:
-        return new GCFProcessor();
+      case Task.ProcessCloudFunctions:
```

Comment:
Just made the naming more consistent here

Reply:
I'll be doing another sweep through the entire project to fix other naming inconsistencies